homeassistant/components/ohme/config_flow.py

```diff
@@ -53,12 +66,22 @@ async def async_step_user(
                 errors["base"] = "invalid_auth"
             except ApiException:
                 errors["base"] = "unknown"
-
-            if not errors:
+            else:
+                if self.source == SOURCE_REAUTH:
+                    return self.async_update_reload_and_abort(
+                        self._get_reauth_entry(),
+                        data_updates=user_input,
+                    )
                 return self.async_create_entry(
                     title=user_input[CONF_EMAIL], data=user_input
                 )
 
         return self.async_show_form(
             step_id="user", data_schema=USER_SCHEMA, errors=errors
         )
+
+    async def async_step_reauth(
+        self, entry_data: Mapping[str, Any]
+    ) -> ConfigFlowResult:
+        """Handle re-authentication."""
+        return await self.async_step_user()
```

Comment:
you'll need to go to `async_step_reauth_confirm` from here first and there use your new schema and go to user or it doesn't do the flow properly in the UI.

Reply:
How would I reuse the user step in that case? I left the REAUTH_SCHEMA in by mistake, I intended to take it out to just reuse the user schema.

As it is now, it seems to work fine in the UI for me, I was following the same approach as other integrations like deluge and picnic.

Reply:
you actually don't need it if you're not passing anything to `asnyc_step_user`. The question is now: Can you change the email in the Ohme app/website?